packages/client/src/pages/ForumTopic/components/ForumAddCommentForm.tsx

```diff
@@ -12,28 +16,49 @@ const ForumAddCommentForm = () => {
   const ref = useRef<HTMLInputElement>(null)
   const [isPickerVisible, setPickerVisible] = useState(false)
   const [inputStr, setInputStr] = useState('')
+  const user = useAppSelector(selectUserInfo)
+  const dispatch = useAppDispatch()
+  const { topicId } = useParams()
 
+  // TODO: баг с курсором
   const handleEmojiClick = (emojiObject: any, event: MouseEvent) => {
     const cursor = (ref?.current && ref.current.selectionStart) ?? 0
     const text = inputStr.slice(0, cursor) + emojiObject.emoji
     setInputStr(text)
   }
 
+  const onSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault()
+    const formData = new FormData(e.currentTarget)
+    let topicIdNumber = 0
+
+    if (topicId) {
+      topicIdNumber = parseInt(topicId)
+    }
```

Comment:
Тут можно тернарный оператор использовать
```suggestion
    const topicIdNumber = topicId ? parseInt(topicId) : 0
```